plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -209,7 +237,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         })->default_value(eosio::chain::config::default_wasm_runtime, default_wasm_runtime_str), wasm_runtime_opt.c_str()
```

Comment:
I suspect the reason you need a `default_wasm_runtime_str` here is because only `operator>>` is defined for vm_type and not a `operator<<` (see libraries/chain/wasm_interface.cpp). I suspect if you were to move the `vm_type_string()` logic over to a `operator<<` there would be no need for `default_wasm_runtime_str`

If so, this is still just a minor issue and not something I think absolutely must be done.